starwars/src/containerChar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Characters from "./characters";
+import styled from "styled-components";
+
+const FlexBox = styled.div`
+  display: grid;
```

Comment:
I see you called the Div FlexBox. Did you mean to use display: flex; ? If not it's okay probably had plans to use flexbox and then later decided for grid but forgot to change the name of the div.